ci-operator/config/kube-reporting/metering-operator/kube-reporting-metering-operator-release-4.6.yaml

```diff
@@ -48,6 +48,22 @@ images:
 - dockerfile_path: olm_deploy/Dockerfile.registry
   from: base
   to: metering-ansible-operator-registry
+operator:
+  bundles:
+  - dockerfile_path: Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/openshift/origin-ghostunnel:4.6
+    with: stable:ghostunnel
+  - pullspec: quay.io/openshift/origin-metering-ansible-operator:4.6
+    with: pipeline:metering-ansible-operator
+  - pullspec: quay.io/openshift/origin-metering-hadoop:4.6
+    with: stable:metering-hadoop
+  - pullspec: quay.io/openshift/origin-metering-hive:4.6
+    with: stable:metering-hive
+  - pullspec: quay.io/openshift/origin-metering-presto:4.6
+    with: stable:metering-presto
+  - pullspec: quay.io/openshift/origin-metering-reporting-operator:4.6
+    with: pipeline:metering-reporting-operator
```

Comment:
For reviewers: https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/help/operators

We use `with: pipeline:*` for any image that is also referenced in the images stanza (i.e. the metering and reporting operators).